src/db-service/UserDbService.js

```diff
@@ -9,6 +10,27 @@ module.exports = {
 	removeUnRegisteredUser: async (fcmToken) => {
 		const response = await User.findOneAndUpdate({fcmToken},{status: 'inactive'},{useFindAndModify: false})
 		return response
+	},
+
+	calculateUserSpecificWeight: async (nid) => {
```

Comment:
can you rename nid to more descriptive word?

Reply:
nid suru bata nai user schema haru ma thiyo so tehi continue gareko, sabai change garam ra?